src/zope/formlib/interfaces.py

```diff
@@ -878,7 +876,7 @@ class IFormField(Interface):
         disambiguate fields with the same name (e.g. from different
         schema) within a collection of form fields.
         """,
-        default="",
+        default="form_field",
```

Comment:
I'm against this change.  If the constraint rejects blank prefixes, that's a bug in the constraint regexp and should be fixed.

Also, I believe the constraint regexp should be updated to allow periods.

Reply:
Fixed in https://github.com/zopefoundation/zope.formlib/pull/35/commits/1c84b39469ab126d4699c875b003f480d5246a1e